#LaneLine message wrt coordinate frame of image where (0,0) is the upper left pixel of the image. m is the slope of the line, b is the y-intercept in y=mx_b form. x is positive as you go toward the right. y is positive as you go down.

Header header

float32 m
float32 b
